snips_nlu/cli/dataset/entities.py

```diff
@@ -56,16 +57,25 @@ def from_file(cls, filepath):
             if six.PY2:
                 it = list(utf_8_encoder(it))
             reader = csv.reader(list(it))
+            autoextent = True
             for row in reader:
                 if six.PY2:
                     row = [cell.decode("utf-8") for cell in row]
                 value = row[0]
+                m = re.match(
+                    r'^#\sautomatically_extensible=(true|false)\s*$',
+                    row[0], re.IGNORECASE)
```

Comment:
This regex should be a global variable so it is created only once instead of creating it at each row:
```
AUTO_EXTENSIBLE_REGEX = re.compile(r"^#\sautomatically_extensible=(true|false)\s*$", re.IGNORECASE)
```

Besides, only the first line should be checked by the regex in order to reduce computation (I'm thinking about big entity files for which this can result in long processing).